grails-app/migrations/0.9.x/changelog-2024-08-05-1300-add-invoice-item-type-to-invoice-item.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="anadolny" id="050820241300-0">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="invoice_item" columnName="invoice_item_type"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="invoice_item">
+            <column name="invoice_item_type" type="CHAR(38)" />
```

Comment:
Is this CHAR(X) what we typically do for our enum fields?

Reply:
I did not notice this CHAR(38) when I picked this up, I think usually we do VARCHAR(255) for enum values, CHAR(38) is for ids - I can change that